api/v3/Case/Getstats.php

```diff
@@ -87,6 +87,10 @@ function civicrm_api3_case_getstats(array $params) {
     $caseTypesParams['id'] = $params['case_type_id'];
     $caseTypes = civicrm_api3('CaseType', 'get', $caseTypesParams);
 
+    if ($caseTypesParams['id']['IS NULL']) {
+      $caseTypes = ['IS NULL'];
+    }
+
```

Comment:
Can we change this to: 
```php
   if (!empty($caseTypesParams['id']['IS NULL'])) {
      $caseTypes =  ['values' => ['0' => ['id' => 'IS NULL']]];
    }
```

Looks like this whole function needs to be refactored but we can do that later.